node/flatpak-node-generator.py

```diff
@@ -690,6 +717,28 @@ def process_lockfile(self, lockfile: Path) -> Iterator[Package]:
         raise NotImplementedError()
 
 
+class RCFileProvider:
+    RCFILE_NAME: str
+
+    def parse_rcfile(self, rcfile: Path) -> Dict[str, str]:
```

Comment:
Should be private I think.

Reply:
Nah, it just parses rcflile to key-value mapping, without anything specific to the generator. It should have kind of "stable api".